db/seeds.rb

```diff
@@ -1,308 +1,74 @@
 # This file should contain all the record creation needed to seed the database with demo values.
 # The data can then be loaded with `rails db:seed` (or along with the creation of the db with `rails db:setup`).
-
 unless Rails.env.development?
   puts "Database seeding has been configured to work only in development mode."
-  return 
+  return
 end
 
-def random_record(klass)
-  # FIXME: This produces a deprecation warning. Could replace it with: .order(Arel.sql('random()'))
-  klass.limit(1).order(Arel.sql('random()')).first
-end
+Dir[
+  File.join(Rails.root, 'db', 'seeders', '**/', '*.rb')
```

Comment:
I suggest removing the trailing slash from `**/`. It's not necessary and might cause portability problems if we should ever run this on Windows.

Reply:
Tks. I didn't know about that.